src/util/getWebviewHtml.ts

```diff
@@ -99,6 +101,8 @@ export async function getWebviewHtml({
   );
   const documentWebviewUri = panel.webview.asWebviewUri(document.uri);
 
+  const diffResults2 = await diffResults;
```

Comment:
Do `await` when setting `diffResults`

Reply:
I am not familiar with TS, and await was prompted by the compiler to be added.